arbor/mechanisms/NaTa_t.mod

```diff
@@ -0,0 +1,53 @@
+NEURON  {
+    SUFFIX NaTa_t
+    USEION na READ ena WRITE ina
+    RANGE gNaTa_tbar
+}
+
+UNITS {
+    (S)  = (siemens)
+    (mV) = (millivolt)
+    (mA) = (milliamp)
+}
+
+PARAMETER {
+    gNaTa_tbar = 0.00001 (S/cm2)
+}
+
+STATE {
+    m
+    h
+}
+
+BREAKPOINT {
+    SOLVE states METHOD cnexp
+    ina = gNaTa_tbar*m*m*m*h*(v-ena)
+}
+
+DERIVATIVE states {
+    LOCAL qt, mAlpha, mBeta, mRate, hAlpha, hBeta, hRate
+    qt = 2.3^((34-21)/10)
+
+    mAlpha = -0.182*6*exprelr(-(v + 38)/6)
```

Comment:
could we write an `m_alpha` function, like this:

```
FUNCTION m_alpha(v) {
    m_alpha = -0.182*6*exprelr(-(v + 38)/6)
}
```
This could then be called in the `DERIVATIVE` and `INITIAL` blocks.
It would reduce the risk of cut and paste errors (and errors when one part of the code is refactored and the other missed).